platform/pulp/app/serializers/task.py

```diff
@@ -94,6 +94,11 @@ class WorkerSerializer(ModelSerializer):
         read_only=True
     )
 
+    online = serializers.BooleanField(
+        help_text='Whether is the worker online or not. Default is True.',
```

Comment:
s/Whether is the worker online or not. Default is/Whether the worker is online or not. Defaults to/